src/questions/questions.controller.ts

```diff
@@ -165,6 +166,7 @@ export class QuestionsController {
       body.content,
       body.type,
       body.topics,
+      this.authService.decode(auth).authorization.userId,
```

Comment:
not verify(auth).userId?

Reply:
[modified](https://github.com/SageSeekerSociety/cheese-backend/pull/48/commits/e915a71ae011a61b6e295abb88755e9776310ccc)